docs/config.md

```diff
@@ -131,3 +135,31 @@ The JSON file has the following fields:
   * `jira`: Optional Jira server and login information
   * `whitesource`: Optional whitesource server authentication information
   * `github_oauth`: Optional project specific GitHub OAuth token
+
+### create findings-[project].yaml files for each project
+
+The `findings-[project].yaml` file (where `[project]` is replaced with the project name used in the config file) contains information used in the generation of the HTML report.  The file contains a list of `findings` with the following fields:
+* `id` Id unique to each finding
+* `priority` Priority of the finding as displayed in the report
+* `text` text used for the finding in the HTML report
+* `title` optional field which, if included, will set the title of the ticket in JIRA
+
+The `findings` elements contains one or more of the following matches fields.  A match is made if ALL of the match conditions are met (e.g. an AND of all matches). 
+* `matches-license` license name that this finding applies to as it appears in the fossology.  Note that the license name will be as it appears after the translations applied after retrieving the SPDX file -- e.g. how it appears on the web report, which might be different from how it appears in Fossology.
+* `matches-path` file path that the finding applies to
```

Comment:
Sorry, I should have noted a few details here also for `matches-path`:
* it will match if the file path in the findings file is a string match to any substring in a file's full path.
* I _think_ that you can put `!` at the front of a `matches-path` entry to make it so that a file will only match if it does _not_ contain that string in its path. If you do this, note that you'll have to surround the string in quotation marks so that it's valid YAML. findings-lfenergy.yaml should have an example of this.
* You can put `$` at the end of the `matches-path` entry which means that it will only match if the string is at the end of the file's path

https://github.com/lfscanning/scaffold/blob/master/findings.py#L67 and subsequent lines contain the parsing for these options.

Reply:
Resolved with the most recent commit